qt_ui/windows/basemenu/QRecruitBehaviour.py

```diff
@@ -136,17 +136,12 @@ def buy(self, unit_type: Type[UnitType]):
         self.update_available_budget()
 
     def sell(self, unit_type):
-        if self.pending_deliveries.units.get(unit_type, 0) > 0:
+        if self.pending_deliveries.units.get(unit_type, 0) > 0 or self.cp.base.total_units_of_type(unit_type) > 0:
```

Comment:
`if pending + available > 0` is clearer I think

Adding

```
@property
def available_next_turn(self, unit_type: Type[UnitType]) -> int:
    ...
```

to the `UnitDeliveryEvent` might simplify a bit?

Reply:
Done, and used this everywhere else it's relevant too.